tests/rados/rados_test_util.py

```diff
@@ -140,7 +140,7 @@ def get_slow_requests_log(node, start_time, end_time, service_name="mon"):
         d_out, d_err = node.exec_command(
             cmd=f"systemctl list-units --type=service | grep ceph | grep {service_name} | head -n 1"
         )
-        daemon = d_out.split(" ")[0].rstrip()
+        daemon = d_out.lstrip().split(" ")[0].rstrip()
```

Comment:
we can just usestrip() method to trim both leading and trailing white spaces, instead of lstrip() & rstrip().